packages/dds/map/src/directory.ts

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+/* eslint-disable tsdoc/syntax */
```

Comment:
Why is this disabled?

Reply:
It is because I added one link in the comments and it uses two at signs

![image](https://github.com/microsoft/FluidFramework/assets/86381878/49edd27f-17b0-4f99-b060-0f741aa0eccd)

Reply:
> You might be able to change the comment to work with the lint rule with some sort of escape sequence (e.g. maybe `\@\@` ?), but not sure. Something to look into.
> 
> If you can't do that, you should definitely go with a single-line disable here: false positives for that lint rule are really rare.

Reply:
> You might be able to change the comment to work with the lint rule with some sort of escape sequence (e.g. maybe `\@\@` ?), but not sure. Something to look into.
> 
> If you can't do that, you should definitely go with a single-line disable here: false positives for that lint rule are really rare.

Using backslash would make the URL incorrect. Instead, I encoded `@` with `%40` and the URL seems to be functioning correctly.